keps/README.md

```diff
@@ -1,7 +1,42 @@
 # Kubernetes Enhancement Proposals (KEPs)
 
-This directory contains both the template and process for KEPs.
+A Kubernetes Enhancement Proposal (KEP) is a way to propose, communicate and coordiante on new efforts for the Kubernetes project.
+You can read the full details of the project in [KEP-1](0001-kubernetes-enhancement-proposal-process.md).
 
-This process is still in an _alpha_ state and is opt-in for those that want to provide feedback for the process.
+This process is still in a _beta_ state and is opt-in for those that want to provide feedback for the process.
 
-See [KEP-1](0001-kubernetes-enhancement-proposal-process.md) for details of this process.
+## Quick start for the KEP process
+
+1. Socialize an idea with a sponsoring SIG.
+   Make sure that others think the work is worth taking up and will help review the KEP and any code changes required.
+2. Follow the process outlined in the [KEP template](0000-kep-template.md)
+
+## FAQs
+
+### Do I have to use the KEP process?
+
+Generally, no!
```

Comment:
I'd like a "No, but... statement"   imo KEPs are pretty low overhead and provide some structure which has been lacking in the community for some time.  I think outlining the benefits would help.